data/podcast_player.json

```diff
@@ -0,0 +1,15 @@
+{
+    "apple": "https://a.fireside.fm/assets/badges/apple_podcasts-ec1be8838a44d6fba0799ec955af4c7e2c1067ec3de32be0e0afe4020c939f29.png",
+    "google_play": "https://a.fireside.fm/assets/badges/google_play-a253eed8c5cda3588a8ccf94edf7ec217bd5122726715c69b052b11376374a8d.png",
+    "google_podcast": "https://a.fireside.fm/assets/badges/google_podcasts-a66d7cc8179da7ad8485d8f41554b2126a2e3891c6a4f9d17b894058e0e21264.png",
```

Comment:
As mentioned [previously](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/291#discussion_r946883932), the Google Play Music link doesn't anymore, so we don't need this icon. 
```suggestion
```